### ISSUE TITLE
Support for Badgr ?

### ISSUE THREAD

**Issue by mboisson** [openedx/credentials#2639]
Hi,
It appears that this plugin is due to replace built-in Badges support in Sumac. However, the Badging feature that was removed from Redwood is for Badgr, which is not supported in this plugin. 

Can it be added to this plugin ? Credly's pricing is not displayed, but any pricing is bad compared to the free option of using Canvas Badges (Badgr) for our non-profit organization, as we are not charging for courses.

**Comment by deborahgu** [comment]
@GlugovGrGlib I think this is a better question for you than for 2U.

**Comment by mboisson** [comment]
Update: just got a call with Credly, and indeed their pricing is out of our budget, while we could use the free tier from Canvas for Badgr.

**Comment by zemogle** [comment]
This is about to kill my OpenEdx project. One of the core features requested was badges. I don't need any universal integration. I would have been happy to add all the badges myself in the Django admin. For an open source project, I find it very frustrating that openedx has gone for Credly which has no free option.

**Comment by deborahgu** [comment]
@e0d flagging this for you.

**Comment by e0d** [comment]
Hi @zemogle .  The legacy Badgr support has some issues.  However, the intention is not to lock users into any particular badging solution.  Rather, it is to have a standard way to push badges from the platform to the preferred wallet of the organization using the platform.

Credly was one of the first adapters created because a particular community member needed it and implemented it.

I suspect that you are not the only user who is interested in Badgr.  Do you have any time or interest in helping to build that?

Anyone else who does, please respond to this thread and hopefully we can get critical mass.

**Comment by mboisson** [comment]
Do you have pointers to the implementation of an "adapter" ? and pointers to the code of how Badgr used to work ?

**Comment by e0d** [comment]
@mboisson I don't think we have great documentation yet as we'e working on the second adapter just now, and doing some refactoring and generalization at the same time.

@cmltaWt0 What do you think is the best place to look to understand what we have and how to approach extending?

**Comment by zemogle** [comment]
I'm trying to customise OpenEdx for my observatory's education programme. I don't think I'm the right person to contribute to this project. I'm totally new to OpenEdx and my brain is already melting.

**Comment by e0d** [comment]
@zemogle understood.  I expect this to get picked up, but when will depend on when a community member can prioritize it.  I'll see if there's an opportunity to drive it forward on our side.

Can you be clear about what you would like to see?  I assume that it's integration with Canvas Badges, formerly Badgr, is that correct?

**Comment by mboisson** [comment]
For me, yes, an integration with Canvas Badges, formerly Badgr. From what I understood discussing with people from Canvas Badges, the API is the same as before, it's just a re-branding that happened, the product has not changed.

**Comment by zemogle** [comment]
For me, I need a simple system to provide badges (which is free). I don't need Badgr/Canvas Badges. Previously I've made a Django app which can provide badges but I have so many other priorities to get our installed to deployment, I can't develop my own right now.
